pkg/proc/amd64util/xsave_amd64.go

```diff
@@ -0,0 +1,30 @@
+package amd64util
+
+import (
+	"sync"
+)
+
+var xstateMaxSize int
+var loadXstateMaxSizeOnce sync.Once
+
+func cpuid(axIn, cxIn uint32) (axOut, bxOut, cxOut, dxOut uint32)
```

Comment:
nit: we could use something like: https://github.com/intel-go/cpuid.

Reply:
In general I think it's easier to read and understand the code as is than to understand the terminology used by that library (the intel manual doesn't use the term "extra features" as far as I can tell). In this particular case, it doesn't seem have one of the things that we need: it doesn't call `CPUID AX=0xd CX=0x0` which returns the size of the XSAVE area.

Reply:
Ack.